src/components/Authenticate/ConsentForm/ClientDescription.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div :class="$style.container" :style="styles.container">
+    <div :class="$style.name">{{ client.name }}</div>
+    <div :class="$style.desc">
+      <div :class="$style.descTitle" :style="styles.descTitle">
+        開発者
+      </div>
+      <div :class="$style.descContent">
+        <template v-if="propst.developer">
+          <user-icon
+            :class="$style.developerIcon"
+            :user-id="propst.developer.id"
+            :fallback-icon-file-id="propst.developer.iconFileId"
+            :size="24"
+          />
+          {{ propst.developer.displayName }} (@{{ propst.developer.name }})
+        </template>
+        <template v-else>
+          <div>=====</div>
+        </template>
+      </div>
+      <div :class="$style.descTitle" :style="styles.descTitle">
+        説明
+      </div>
+      <div :class="$style.descContent">
+        {{ client.description }}
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, reactive, PropType } from '@vue/composition-api'
+import { makeStyles } from '@/lib/styles'
+import { OAuth2Client, User } from '@traptitech/traq'
+import UserIcon from '@/components/UI/UserIcon.vue'
+
+const useStyles = () =>
+  reactive({
+    container: makeStyles(theme => ({
+      background: theme.background.secondary,
+      color: theme.ui.secondary
```

Comment:
```suggestion
      color: theme.ui.primary
```
コンテンツの主たる部分なのでprimaryのほうがいいかなと (`.descTitle`がboldなので内容がわかりづらい)